website/templates/website/header_footer.html

```diff
@@ -45,6 +45,7 @@
             <a href="#">Student Services</a>
             <ul id="subnav-2" class="subnav">
                 <li><a href="/oh">Office Hours</a></li>
+                <li><a href="/events">Past Events</a></li>
```

Comment:
i think it would be better to have this url be `pastevents` or `past_events` to avoid confusion